packages/css-data/src/shorthands.test.ts

```diff
@@ -1448,3 +1448,77 @@ test("expand position-try", () => {
     ["position-try-options", "--dashed-ident"],
   ]);
 });
+
+test("Does not fail on empty value", () => {
```

Comment:
is there any way we could get a full list of all properties  as an array and get a loop over it, then add a bunch of values beyond empty string to get generated tests for all of them?

Reply:
I don't think its feasible to maintan a list of properties here